code/upstream-integration-tests/src/androidTest/java/com/adobe/marketing/mobile/edge/integration/UpstreamIntegrationTests.kt

```diff
@@ -279,26 +282,27 @@ class UpstreamIntegrationTests {
         Edge.sendEvent(experienceEvent) {}
 
         // Verify
-        val expectedLocationHintJSON = """
+        val expectedLocationHint = """
         {
           "payload": [
             {
               "ttlSeconds" : 123,
               "scope" : "EdgeNetwork",
-              "hint" : "stringType"
+              "hint" : "STRING_TYPE"
             }
           ]
         }
-        """.trimIndent()
+        """
 
         // Unsafe access used since testSendEvent_receivesExpectedEventHandles guarantees existence
         val locationHintResult = TestSetupHelper.getEdgeEventHandles(expectedHandleType = TestConstants.EventSource.LOCATION_HINT_RESULT)
             .first()
 
-        JSONAsserts.assertTypeMatch(
-            expected = JSONObject(expectedLocationHintJSON),
-            actual = JSONObject(locationHintResult.eventData),
-            exactMatchPaths = listOf("payload[*].scope")
+        assertTypeMatch(
```

Comment:
As discussed, we can do assertExactMatch here and typeMatch the "payload[*].ttlSeconds".

Reply:
I am fine with this.